devices/bmxx80/bmxx80.go

```diff
@@ -514,6 +514,8 @@ var defaults = Opts{
 	Humidity:    O4x,
 }
 
+var doSleep = time.Sleep
```

Comment:
why are we using `sleep` and `doSleep`, shouldn't we be consistent?

Reply:
There's already a package global sleep variable and I didn't feel like renaming it.